neurodamus/utils/progressbar.py

```diff
@@ -122,14 +122,6 @@ def iter(cls, iterable, end=None, start=0, **kw):
         # __call__ 'end' cant be False -> None
         return cls(end, start, **kw)(iterable, end or None, start)
 
-    @classmethod
-    def itervalues(cls, iterable, **kw):
-        return cls.iter(iterable.values(), len(iterable), **kw)
-
-    @classmethod
-    def iteritems(cls, iterable, **kw):
-        return cls.iter(iterable.items(), len(iterable), **kw)
-
```

Comment:
I think at least one of this is used in multiscale run:
https://github.com/openbraininstitute/MultiscaleRun/blob/45c493da4e6c9ec604c672864b6927a3b5e34977/multiscale_run/simulation.py#L211

Reply:
I don't see any usage of `itervalues` or `iteritems`; the `__iter__` is still there, though.

Reply:
ah in this case, forget this comment